docs/serving/using-knative-services.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "User-created Knative Services"
+weight: 01
+linkTitle: "User-created Knative Services"
+type: "docs"
+---
+
+Knative services are Kubernetes Services that a user creates to deploy an application. Each Knative Service is defined by a Route and a Configuration, which have the same name as the Service, contained in a YAML file.
```

Comment:
This first sentence is not quite right. Knative Services are backed by multiple Kubernetes "Services". This is because the Kubernetes Service are essentially load balancers over a Deployment. See https://kubernetes.io/docs/concepts/services-networking/service/ 

The Knative Service attempts to model a more logical definition of a Service or Application. To do so it has the concept of Revisions which are each backed by a deployment with 2 K8s Services.